helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -157,6 +161,31 @@
     }
   }
 
+  /**
+   * Return first found non-loopback ip from enumeration of network devices.
+   * InetAddress.getLocalHost() will not work, as it returns the loopback device
+   */
+  public static String findFirstInet4Addr(Enumeration<NetworkInterface> nics)
+      throws HeliosDeploymentException {
+    List addrs = new ArrayList<InterfaceAddress>();
+    for (NetworkInterface nic : Collections.list(nics)) {
+      addrs.addAll(nic.getInterfaceAddresses());
+    }
+    return findFirstInet4Addr(addrs);
+  }
+
+  public static String findFirstInet4Addr(Collection<InterfaceAddress> addrs)
+      throws HeliosDeploymentException {
+    for(InterfaceAddress addr : addrs) {
+      String addrStr = addr.getAddress().toString();
+      if(addrStr.matches("^/\\d+\\.\\d+\\.\\d+\\.\\d+$") &&
+          !addrStr.startsWith("/127")) {
```

Comment:
I would suggest using `interfaceAddress.getAddress().isLoopbackAddress()` instead of testing against `/127`